systemlinkmigrate.py

```diff
@@ -1,17 +1,29 @@
 # Generic migration utility for migrating various data and settings between SystemLink servers.
 # Not all services will be supported. Addtional services will be supported over time.
 
-import sys
+import sys, os
 from slmigrate import mongohandler, filehandler, arghandler, servicemgrhandler, constants
```

Comment:
Let's follow PEP 8 guidance on imports.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
From looking at how the build is configured I believe we need to change the follwing to remove the `exit-zero` argument. 

```yaml
    - name: Lint with flake8
      run: |
        # stop the build if there are Python syntax errors or undefined names
        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics --exclude basic_scripts
        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics --exclude basic_scripts
```

Reply:
Shouldn't the first rune of flake catch any errors? I may be reading this wrong.